src/Pages/jiminkim/Main/MainRight/MainRight.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { INFO } from '../../footerData.js';
+import ProfileBox from './ProfileBox/ProfileBox';
+
+export default class MainRight extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      storyProfileList: [],
+      recommendProfileList: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/jiminkim/profileData.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
```

Comment:
```suggestion
    fetch('/data/jiminkim/profileData.json')
      .then(res => res.json())
```
- public 경로에서 부를 경우 위와 같이 생략가능합니다
- get method도 기본값이기 때문에 생략가능합니다

Reply:
```
    fetch('/data/jiminkim/feeds.json')
      .then(res => res.json())
      .then(data => {
        this.setState({
          feeds: data.feeds,
        });
      });
```
훨씬 간결해 지네요! 감사합니다.